docs/how-to/logs.rst

```diff
@@ -0,0 +1,56 @@
+==============
+Accessing logs
+==============
+
+The :ref:`MicroOVN services` provide logs as part of their normal operation.
+
+By default they are provided through the systemd journal.
+
+Below you can view an example for how to access the logs of the
```

Comment:
Just for my curiosity, is the phrase `This is how to ...` more standard across our documentation? I'm just asking because I don't see much difference between the two, `Below you can ...` has even fewer words :laughing:

Reply:
It would be:

"This is how to access the logs ..."

Reply:
Oh, of course, I didn't realize that it'll alter the sentence like this. Thanks